modules/issue_tracker/jsx/IssueForm.js

```diff
@@ -441,7 +445,7 @@ class IssueForm extends Component {
         response.json().then((data) => {
           this.setState({submissionResult: 'error'});
           let msgType = 'error';
-          let message = data.message || 'Failed to submit issue :(';
+          const message = data.error ?? data.message;
```

Comment:
This change makes it so the user actually sees the error message if one happens. Example visit needs a PSCID when submitting issue with visit.